### PR TITLE
Values method for face quad interpolator 

### PR DESCRIPTION
**Pull request by artv3**
Shortcut for users who only need values. 

 | PR | Author | Editor | Reviewers | Assignment | Approval | Merge| 
 | --- | --- | --- | --- | --- | --- | --- | 
| [#1365](https://github.com/mfem/mfem/pull/1365) | @artv3 | @tzanio | @camierjs + @YohannDudouit | 03/19/20 | 03/20/20 | 03/25/20 | |

### CODE REVIEW

File: fem/quadinterpolator_face.cpp [hunk 1]
```diff
@@ -495,4 +495,11 @@ void FaceQuadratureInterpolator::Mult(
    }
 }
 
+void FaceQuadratureInterpolator::Values(
+   const Vector &e_vec, Vector &q_val) const
+{
+   Vector q_der, q_det, q_nor;
+   Mult(e_vec, 0x1, q_val, q_der, q_det, q_nor);
```
Comment (v-dobrev) [inline fem/quadinterpolator_face.cpp]: Use the enum constant here.

**Comment by tzanio** [comment]
Merged in `next` for testing...

**Comment by camierjs** [comment]
I restarted the travis jobs, two last ones were hanging.

File: fem/quadinterpolator_face.cpp [hunk 2]
```diff
@@ -495,4 +495,11 @@ void FaceQuadratureInterpolator::Mult(
    }
 }
 
+void FaceQuadratureInterpolator::Values(
+   const Vector &e_vec, Vector &q_val) const
+{
+   Vector q_der, q_det, q_nor;
+   Mult(e_vec, VALUES, q_val, q_der, q_det, q_nor);
```
Comment (artv3) [inline fem/quadinterpolator_face.cpp]: @YohannDudouit , @v-dobrev  using VALUES doesn't work as expected. Is that not the enum constant?

Comment (artv3) [reply fem/quadinterpolator_face.cpp]: Ah hang on, I think I might have a different bug in my application.

Comment (artv3) [reply fem/quadinterpolator_face.cpp]: nvm, it works great.

**Comment by tzanio** [comment]
Seems like #1368 needs more discussion. This one is simple, so I'll merge it directly...

**Pull request merged**